ct-aa/dotTheory---JavaScript
# ✨ JS-614: Review Adding Multiple Routes

# Related Tickets
- [JS-614](https://codeandtheory.atlassian.net/browse/JS-614)

# Description
Added the **Review Adding Multiple Routes** challenge solutions.

# Validation
* Open the Terminal and go to the directory of the **Review Adding Multiple Routes** folder.
* Run "node app.js".
* Visit http://localhost:3000/blog on the browser.
* The browser should display the sample posts.

<img width="1779" alt="Screen Shot 2023-01-24 at 3 29 38 PM" src="https://user-images.githubusercontent.com/93116252/214234844-11fa5bf0-b1e1-43cd-820b-236892942579.png">

# Risks
N/A
